src/util/system.h

```diff
@@ -294,15 +294,13 @@ class ArgsManager
      * Get data directory path
      *
      * @return Absolute path on success, otherwise an empty path when a non-directory path would be returned
```

Comment:
27176: we didn't backport 27073, which is dependency for this backport

Reply:
dropped